src/clj/rems/db/outbox.clj

```diff
@@ -17,7 +18,7 @@
    :outbox/deadline DateTime
    :outbox/next-attempt (s/maybe DateTime)
    :outbox/latest-attempt (s/maybe DateTime)
-   :outbox/latest-error (s/maybe s/Str)
+   :outbox/latest-error (s/maybe s/Any)
```

Comment:
is the error from entitlement-push structured?

Reply:
Yes the push actually just returns the result of the HTTP call as the data. I think this was mainly done for better testing, but it'll provide more debugging possibilities for the outbox with the richer data stored there. Since it's a JSON blob anyway, I don't see the serialization to a string useful.